api/konnect/v1alpha1/konnect_conditions.go

```diff
@@ -214,3 +214,18 @@ const (
 	// condition type indicating that the Konnect network reference is invalid.
 	KonnectNetworkRefsReasonInvalid = "Invalid"
 )
+
+const (
+	// ControlPlaneMirroredConditionType is the type of the condition that indicates
+	// the status for the control plane mirroring.
+	ControlPlaneMirroredConditionType = "Mirrored"
+
+	// ControlPlaneMirroredSucceededConditionReason is the reason used with the
+	// ControlPlaneMirroring condition type indicating that the control plane
+	// mirroring has succeeded.
+	ControlPlaneMirroredSucceededConditionReason = "Mirrored"
+	// ControlPlaneMirroredFailedConditionReason is the reason used with the
+	// ControlPlaneMirroring condition type indicating that the control plane
+	// mirroring has failed.
+	ControlPlaneMirroredFailedConditionReason = "Failed"
```

Comment:
```suggestion
	// ControlPlaneMirroredSucceededConditionReason is the reason used with the
	// Mirrored condition type indicating that the control plane mirroring has succeeded.
	ControlPlaneMirroredSucceededConditionReason = "Mirrored"
	// ControlPlaneMirroredFailedConditionReason is the reason used with the
	// Mirrored condition type indicating that the control plane mirroring has failed.
	ControlPlaneMirroredFailedConditionReason = "Failed"
```

Reply:
What's the change here? I can't figure it out 🤔

Reply:
GitHub messes up my diffs.

I've changed `ControlPlaneMirroring` into `Mirrored` since the former is not defined anywhere.

Reply:
👍🏻